### PR TITLE
Karmel - initial PR

### PR DESCRIPTION
**Pull request by karmeli87**

### CODE REVIEW

File: Scenarios/CorruptedCasino/Casino.cs [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+{
+    public class Casino : BaseTest
+    {
+        public static readonly Casino Instance;
+
+        private static bool Local = false;
+
+        static Casino()
+        {
+            var url = Local ? "http://localhost:8080" : "http://10.0.0.69:8090";
```
Comment (aviviadi) [inline Scenarios/CorruptedCasino/Casino.cs]: please pass to here argv[0] instead of url

File: Scenarios/CorruptedCasino/Casino.cs [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+            await ConfigureExpiration();
+            await ConfigureRevisions();
+        }
+
+        private static async Task ConfigureExpiration()
+        {
+            await Store.Maintenance.SendAsync(new ConfigureExpirationOperation(new ExpirationConfiguration
+            {
+                Disabled = false,
+                DeleteFrequencyInSec = 600
```
Comment (aviviadi) [inline Scenarios/CorruptedCasino/Casino.cs]: can be random (using BaseTest's Random instance), and logged with ReportInfo

File: Scenarios/CorruptedCasino/Casino.cs [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+                        // expected
+                    }
+                    catch (JavaScriptException e) when(e.Message.Contains("Lottery is over!"))
+                    {
+                        // expected
+                    }
+                    catch (Exception e)
+                    {
+                        // not expected at all!
+                        Console.WriteLine(e);
```
Comment (aviviadi) [inline Scenarios/CorruptedCasino/Casino.cs]: do not WriteLine, nor throw.. ReportFailure with exception instead and exit

File: Scenarios/CorruptedCasino/Casino.cs [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+            var policy = Policy.Handle<TimeoutException>().Retry(5);
+            Console.Write("Creating Lottery ... ");
+            var lottery = await Lottery.CreateLottery();
+
+            Instance.ReportEvent(new EventInfo
+            {
+                Message = $"Lottery {lottery.Id} was created and will overdue at {lottery.DueTime}",
+                Type = EventInfo.EventType.Info
+            });
+            Console.WriteLine($"Done {lottery.Id}");
```
Comment (aviviadi) [inline Scenarios/CorruptedCasino/Casino.cs]: No writeLines..

File: Scenarios/CorruptedCasino/Casino.cs (hunk 3 above)
Comment (karmeli87) [reply Scenarios/CorruptedCasino/Casino.cs]: This is for debugging my own code :) I report failure down the line...
Can you redirect it to dev/null if needed?

File: Scenarios/CorruptedCasino/Casino.cs (hunk 2 above)
Comment (karmeli87) [reply Scenarios/CorruptedCasino/Casino.cs]: Not everything has to be random.

**Pull request merged**